src/content/fr/2019/seo.md

```diff
@@ -0,0 +1,421 @@
+
+L'analyse n'a pas vérifié la bonne mise en œuvre, par exemple si les différentes versions linguistiques se lient correctement les unes aux autres. Cependant, en examinant la faible adoption d'une version x-default (seulement 3,77% sur ordinateur et 1,30% sur mobile), [comme cela est recommandé](https://support.google.com/webmasters/answer/189077?hl=fr), c'est un indicateur que cet élément est complexe et pas toujours facile à bien faire.
+
+### Exploration des SPA (Single Page Application)
+
+Les applications monopages (SPA) construites avec des frameworks comme React et Vue.js ont leur propre complexité SEO. Les sites web utilisant une navigation basée sur le hachage, rendent particulièrement difficile pour les moteurs de recherche de les explorer et de les indexer correctement. Par exemple, Google avait une solution de contournement "AJAX crawling scheme" qui s'est avérée complexe pour les moteurs de recherche ainsi que pour les développeurs, elle a donc été [déconseillée en 2015](https://webmasters.googleblog.com/2015/10/deprecating-notre-ajax-crawling-scheme.html).
+
+Le nombre de SPA testés avait un nombre relativement faible de liens servis via des URL de hachage, avec 13,08% des pages mobiles React utilisant des URL de hachage pour la navigation, 8,15% des pages mobiles Vue.js les utilisant et 2,37% des pages angulaires mobiles les utiliser. Ces résultats étaient également très similaires pour les pages de bureau. Cela est positif à voir du point de vue du référencement, compte tenu de l'impact que les URL de hachage peuvent avoir sur la découverte de contenu.
+
+Le nombre plus élevé d'URL de hachage dans les pages React est surprenant, en particulier contrairement au nombre plus faible d'URL de hachage trouvées sur les pages angulaires. Les deux frameworks favorisent l'adoption de packages de routage où [API historique](https://developer.mozilla.org/en-US/docs/web/API/History) est la valeur par défaut pour les liens, au lieu de s'appuyer sur des URL de hachage. Vue.js [envisage de passer à l'utilisation de l'API Historique par défaut](https://github.com/vuejs/rfcs/pull/40) ainsi que dans la version 3 de leur package `vue-router`.
+
+### AMP
+
+AMP (Pages mobiles accélérées) a été introduit pour la première fois en 2015 par Google en tant que framework HTML open source. Il fournit des composants et une infrastructure aux sites web pour offrir une expérience plus rapide aux utilisateurs, en utilisant des optimisations telles que la mise en cache, le chargement différé et des images optimisées. Notamment, Google a adopté cela pour son moteur de recherche, où les pages AMP sont également servies à partir de leur propre CDN. Cette fonctionnalité est devenue plus tard une proposition de normes sous le nom [échanges HTTP signés](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html).
```

Comment:
Do we need English definition of AMP? Or is French sufficient even though it doesn't match the acronym? Not sure what the norm is here. Wikipedia puts it in English: https://fr.wikipedia.org/wiki/Accelerated_Mobile_Pages

Reply:
i don't think we do need it. Since in French AMP is known as accelerated mobile pages. Do you think we should add a link to Wiki page on AMP anchor ?

Reply:
I think it's OK without the wikipedia page. It is explained in the chapter text. If anything we should link to https://amp.dev/. However, if we're doing that we should add to French, English and Japanese versions.